ui/helpers/utils/util.js

```diff
@@ -326,12 +329,14 @@ export function addHexPrefixToObjectValues(obj) {
  * Given the standard set of information about a transaction, returns a transaction properly formatted for
  * publishing via JSON RPC and web3
  *
- * @param {boolean} [sendToken] - Indicates whether or not the transaciton is a token transaction
- * @param {string} data - A hex string containing the data to include in the transaction
- * @param {string} to - A hex address of the tx recipient address
- * @param {string} from - A hex address of the tx sender address
- * @param {string} gas - A hex representation of the gas value for the transaction
- * @param {string} gasPrice - A hex representation of the gas price for the transaction
+ * @param {object} options
+ * @param {boolean} [options.sendToken] - Indicates whether or not the transaciton is a token transaction
```

Comment:
Yes - the brackets are how optional parameters are the JSDoc-style way of denoting optional parameters. These are omitted in TypeScript repositories because the type explains whether it's optional or not, but we still need some indicator in JavaScript.

Reply:
Ah ok I figured something like that! That makes sense.